app/views/user_mailer/activation_invite.html.erb

```diff
@@ -7,5 +7,6 @@
     <h2>Welcome to BattleShift!</h2>
     <p>Thanks for signing up <%= @user.name %>!</p>
     <%= link_to "Click here to activate your account", @url %>
+    <p>Your api key is: <%= @user.id %>, please keep it secret.</p>
```

Comment:
Can we add "Forward this email to ten friends or hailstones the size of destroyers will rain down on your car" to the end?

Reply:
Absolutely I meant to do that